content/documentation/overview/content.md

```diff
@@ -0,0 +1,40 @@
++++
+fragment = "content"
+weight = 10
+
+title = "Overvew"
+
+[sidebar]
+  sticky = true
++++
+
+<p align="center" style="padding-top: 1em; padding-bottom: 1em">
+  <img src="/images/diagramanimated.gif" width="60%" style="border-radius: 5px" />
+</p>
+
+### Why do I need CDT.cloud?
+
+CDT.cloud provides business-friendly open source software components for building custom web-based C/C++ development tools, running in the cloud or on the desktop.
+Such custom development tools make it easy for your users to work with specific devices, toolchains, or libraries, etc.
+With a custom tool that is optimized for a specific purpose, you can achieve a great developer experience for your users by including, for instance, ...
+
+* wizards for creating projects and and custom editors for specific configuration artifacts
+* example projects to get users up and running quickly
+* excellent language editing support for C/C++ and other languages without manual user configuration
+* easy-to-use debug configurations or other analysis capabilities, such as memory inspectors or trace viewers,
+* and much more.
+
+### How does CDT.cloud help?
+
+Custom C/C++ development tools have several reoccuring capabilities, such as creating projects, editing source code, building projects, debugging and tracing applications, etc.
+While they may differ from one specific tool use case to another, the base components can still be reused and adapted to the specifics of an environment.
+This makes building such tools significantly more efficient.
+And that's precisely where CDT.cloud is here to help: it provides a feature-rich ecosystem of reusable platforms and components that can be customized for your environment and bundled into a ready-to-use tool product for your use case.
+
+### How do I get started?
+
+In most of the cases, a great starting point is to selecting a [development tool platform]({{< relref  "tool-platform" >}}) for your custom tool, which you augment with VS Code extensions too add required tool capabilities and then put customizations for your specific environment on top.
```

Comment:
When I read this as a novice I don't know what is meant with "development tool platform". Clicking on the link also only refers to Theia, so I still don't know what any alternative could be. Maybe just remove this sentence completely and just start with the Blueprint?